src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/HomePage.java

```diff
@@ -0,0 +1,171 @@
+package com.marizueva.laboratory.hw3.fluentpageobject.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends CommonPageElements {
+
+
+    @FindBy(xpath = "//*[@class='benefit-icon'][1]")
+    WebElement displayedBenefitImage1;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-custom'][1]")
+    WebElement displayedBenefitImage2;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-multi'][1]")
+    WebElement displayedBenefitImage3;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-base'][1]")
+    WebElement displayedBenefitImage4;
+
+    @FindBy(xpath = "(//*[@class='benefit-txt'])[1]")
+    WebElement benefitText1;
+
+    @FindBy(xpath = "(//*[@class='benefit-txt'])[2]")
+    WebElement benefitText2;
+
+    @FindBy(xpath = "(//*[@class='benefit-txt'])[3]")
+    WebElement benefitText3;
+
+    @FindBy(xpath = "(//*[@class='benefit-txt'])[4]")
+    WebElement benefitText4;
+
+    @FindBy(xpath = "//div[@class='main-content']/h3[@class='main-title text-center']")
+    WebElement mainContentHeader;
+
+    @FindBy(xpath = "//div[@class='main-content']/p")
+    WebElement mainContentText;
+
+    @FindBy(xpath = "//iframe[@id='jdi-frame-site']")
+    WebElement iframeCenter;
+
+    @FindBy(xpath = "//img[@src='images/Logo_Epam_Color.svg']")
+    WebElement epamLogoInIframe;
+
+    @FindBy(xpath = "//div[@class='main-content']/h3[@class='text-center']")
+    WebElement subHeaderJdi;
+
+
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        //        commonPageElements = new CommonPageElements(driver);
+    }
+
+    //    public CommonPageElements getCommonPageElements() {
+    ////        return commonPageElements;
+    //    }
+
+    public HomePage openSite() {
+        driver.navigate().to(siteUrl);
+        return this;
+    }
+
+    //    public String getBrowserTitle() {
+    //
+    //        return ;
+    //    }
```

Comment:
Please delete commented code if it does not need

Reply:
fixed